storage/backup/backup-cli/src/storage/command_adapter/mod.rs

```diff
@@ -149,4 +153,20 @@ impl BackupStorage for CommandAdapter {
             .err_notes((file!(), line!(), &buf))?;
         Ok(buf.lines().map(str::to_string).collect())
     }
+
+    async fn backup_metadata_file(&self, file_handle: &FileHandleRef) -> Result<()> {
+        // extract the file name from the file_handle
+        let name = Path::new(file_handle)
+            .file_name()
```

Comment:
Are we expecting it to be prefixed by "metadata/"? Assert and/or document.

Reply:
added document